operator/pkg/lbipam/lbipam.go

```diff
@@ -939,7 +936,7 @@ func (ipam *LBIPAM) findRangeOfIP(sv *ServiceView, ip net.IP) (lbRange *LBRange,
 		return r, false, nil
```

Comment:
shouldn't we return true for `foundPool` here too?
```suggestion
		return r, foundPool, nil
```

Reply:
might be more a question for @dylandreimerink - which might be discussed/solved in the corresponding PR? https://github.com/cilium/cilium/pull/23297

but i'm happy to fix it here as well if it's the expected behaviour.